### ISSUE TITLE
Apps Unsearchable (Windows 10)

### ISSUE THREAD

**Issue by FallingStraw** [memstechtips/UnattendedWinstall#103]
after I installed windows all of the apps I installed cannot be searched and doesn't show anything when I search the apps

**Comment by CoreyKovalik** [comment]
Having the same issue. Have tried re indexing. Also the start menu is clearly in the search

**Comment by rurre** [comment]
> This is because WSearch service is set to manual

Setting it to automatic doesn't seem to change anything. Is cortana required perhaps?

**Comment by jbae-89** [comment]
I am having the same issue.
Tested with Windows 10 on two separate installs, one with IoT-LTSC-Like and one with Standard.

I have tried
- installing all updates
- installing Cortana
- setting WindowsSearch service to automatic
- rebuilding indexes
- Microsoft Search and Indexing Troubleshooter
and a few other recommend tweaks from the MS forums, nothing seemed to fix it.

It looks like 'SearchUI.exe' is missing from the system, I am not sure where that originates or what would remove it but it seems to be necessary for the search functionality to work as intended.

**Comment by vonguyenkhang** [comment]
Can each of the users attach their own XML file which is used during the installation? Thanks.

**Comment by rurre** [comment]
> Can each of the users attach their own XML file which is used during the installation? Thanks.

As mentioned above it happens with both standard and IoT-LTSC-Like. I personally used IoT-LTSC-Like, so you can just look at the one this repo.

**Comment by vonguyenkhang** [comment]
> > Can each of the users attach their own XML file which is used during the installation? Thanks.
> 
> As mentioned above it happens with both standard and IoT-LTSC-Like. I personally used IoT-LTSC-Like, so you can just look at the one from this repo.

Thanks for the reply. I ask that question because the owner of this repo doesn't include any version number or Release snapshot ; any additions or modifications to the template files are immediately available to the users, so it's not easy to have a common understanding of what exact tweaks have been applied.

About the issue with search, I suspect it has to do with the section _Powershell Script to Run Various Tweaks, Most of These are from the Chris Titus Tech Windows Utility (Credit:https://christitus.com/windows-tool/)_ and the following sections which make numerous changes to Services and Registry. Of course there's also a possibility that it's a Windows bug.

Although some of those tweaks seem useful to me, there are also a few others with questionable impact on user experience (eg. Disable Windows Spotlight, Disables Mouse Acceleration, Defer non-security updates - this means no QoL fixes, Disables Storage Sense, etc.). Personally, I prefer those tweaks to be selectable according to each individual's need, but the owner of this project may prefer to have the high-hand approach to get a more appealing results (eg. lower RAM usage in Task Manager, more features to write in the repo summary, etc.).

The more tweaks are applied, the higher the chance of something getting broken, because no one really know what are the underlying dependencies, and everyone have different hardware configurations and usage scenarios.

In conclusion, I suggest removing the previously mentioned tweak sections to see if the problem still persists. As for myself, I'll be following this https://schneegans.de/windows/unattend-generator/ (thanks to @jbae-89 for sharing) and will only sparingly apply the tweaks I actually need. There's no point in saving a few seconds of boot time or a few megabytes of RAM just to boast, especially if it results in a broken machine.

**Comment by thomaschampagne** [comment]
Same issue for me.
Didn't find solution yet

**Comment by ekungurov** [comment]
@thomaschampagne Have you tried enabling Windows Search Service and removing Start_TrackProgs entity from registry?

**Comment by thomaschampagne** [comment]
> @thomaschampagne Have you tried enabling Windows Search service and removing Start_TrackProgs entity from registry?

I tried, it doesn't solves the issue.

**Comment by ekungurov** [comment]
@thomaschampagne thank you for confirming. Please could you also check that Indexing Options window includes "Start Menu" item. Indexing Options could be opened via "control srchadmin.dll" command.

**Comment by thomaschampagne** [comment]
> @thomaschampagne thank you for confirming. Please could you also check that Indexing Options window includes "Start Menu" item. Indexing Options could be opened via "control srchadmin.dll" command.

Yes I confim you:

![image](https://github.com/user-attachments/assets/9d6d7158-1647-4609-beb6-5067ab8d44a1)

@FallingStraw Why closing this issue. It's solved?
